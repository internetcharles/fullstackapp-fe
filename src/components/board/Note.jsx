import React from 'react';
import PropTypes from 'prop-types';

export const Note = ({ title, note }) => (
  <div>
    <h3>{title}</h3>
    <p>{note}</p>
  </div>
);

Note.propTypes = {
  title: PropTypes.string.isRequired,
  note: PropTypes.string.isRequired
};

export default Note;
