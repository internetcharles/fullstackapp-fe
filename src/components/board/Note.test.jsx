import React from 'react';
import { render } from '@testing-library/react';
import Note from './Note';

describe('Note component', () => {
  it('renders note', () => {
    const { asFragment } = render(<Note 
      title="Cheese"
      note="Cheese"
    />);
    expect(asFragment()).toMatchSnapshot();
  });
});
