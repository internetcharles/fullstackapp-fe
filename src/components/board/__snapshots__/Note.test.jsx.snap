// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Note component renders note 1`] = `
<DocumentFragment>
  <div>
    <h3>
      Cheese
    </h3>
    <p>
      Cheese
    </p>
  </div>
</DocumentFragment>
`;
